src/containers/TwinPeaksQuote.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import Header from '../components/Header';
+import Load from '../components/quote/Load';
+import { makeRequest } from '../services/twinPeaksApi';
+
+export default function TwinPeaksQuote() {
+  
+  const [quote, setQuote] = useState();
+
```

Comment:
you might want a piece of loading state here in case your fetch fails, your app won't blow up.